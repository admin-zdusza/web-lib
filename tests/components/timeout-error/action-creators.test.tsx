import {
  timeoutAcknowledged,
  timeoutOccured,
} from "../../../src/components/timeout-error/actions-creators";

it("should create timeout error occured", async () => {
  expect(timeoutOccured()).toMatchSnapshot();
});

it("should create timeout error acknowledged", async () => {
  expect(timeoutAcknowledged()).toMatchSnapshot();
});
