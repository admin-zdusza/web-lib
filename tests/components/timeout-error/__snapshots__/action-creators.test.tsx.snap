// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create timeout error acknowledged 1`] = `
Object {
  "type": "TIMEOUT_ACKNOWLEDGED",
}
`;

exports[`should create timeout error occured 1`] = `
Object {
  "type": "TIMEOUT_OCCURED",
}
`;
